### PR TITLE
#838-RogerHuang-Add Car0 on Gantt Page

### LINKED ISSUE #838: [Frontend] - Add 0 to Car Number Filter on Gantt Page
### Description

Currently, the Car Number filters on the Gantt Page only include 1 and 2 as options. 0 needs to be added as an option to account for projects with Car Number 0.

### Acceptance Criteria

The Car Number filters should include 0, 1, and 2.

### Proposed Solution

Follow the pattern used for Car Numbers 1 and 2 and add a button and the relevant code additions for Car Number 0 in `GanttPageWrapper.tsx` and `GanttPageFilter.tsx`.

### Mocks

Screenshot 2023-02-07 122612

### PR DESCRIPTION
**Pull request by rhuang772**
## Changes

Added an option to select projects tagged as Car Number 0. 

## Notes

There is no mui icon available for 0. (LooksZeroOutlined does not exist in the @mui/icons-material package). To remedy this, 
I changed all the icons for them to Boxes and styled them based on whether they have been checked or not. A ~~green~~ `#ef4345` coloring means it has been checked off.

## Screenshots

Screenshot 2023-02-18 125053

## Checklist

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #838

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -131,5 +131,8 @@
     ]
   },
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
```
Comment (Ari-S-123) [inline package.json:134]: I'm not sure this ticket requires any changes to `package.json`

File: src/frontend/src/pages/GanttPage/GanttPageFilter.tsx [hunk 2]
```diff
@@ -4,10 +4,10 @@
  */
 
 import { WbsElementStatus } from 'shared';
-import LooksOneOutlinedIcon from '@mui/icons-material/LooksOneOutlined';
-import LooksTwoOutlinedIcon from '@mui/icons-material/LooksTwoOutlined';
+//import LooksOneOutlinedIcon from '@mui/icons-material/LooksOneOutlined';
```
Comment (Ari-S-123) [inline src/frontend/src/pages/GanttPage/GanttPageFilter.tsx:7]: You should remove commented-out code.

File: src/frontend/src/pages/GanttPage/GanttPageFilter.tsx [hunk 3]
```diff
@@ -47,16 +49,88 @@ const GanttPageFilter: FC<GanttPageFilterProps> = ({
...
         Car
       </Grid>
-      <Grid item container xs={12} md={1}>
+      <Grid item container xs={10} md={1} sx={{ flexWrap: 'nowrap' }}>
         <Grid item>
           <Checkbox
             defaultChecked
-            icon={<LooksOneOutlinedIcon />}
-            checkedIcon={<LooksOneOutlinedIcon />}
+            //icon={<FiberManualRecordOutlinedIcon />}
```
Comment (Ari-S-123) [inline src/frontend/src/pages/GanttPage/GanttPageFilter.tsx:60]: Here too.

File: src/frontend/src/pages/GanttPage/GanttPageFilter.tsx [hunk 4]
```diff
@@ -47,16 +49,88 @@ const GanttPageFilter: FC<GanttPageFilterProps> = ({
...
+                fontSize: '2rem'
+              },
+              paddingLeft: 0
+            }}
+          />
+        </Grid>
+        <Grid item>
+          <Checkbox
+            defaultChecked
+            //icon={<LooksOneOutlinedIcon />}
```
Comment (Ari-S-123) [inline src/frontend/src/pages/GanttPage/GanttPageFilter.tsx:105]: And here.

File: src/frontend/src/pages/GanttPage/GanttPageFilter.tsx [hunk 5]
```diff
@@ -73,8 +147,35 @@ const GanttPageFilter: FC<GanttPageFilterProps> = ({
         <Grid item>
           <Checkbox
             defaultChecked
-            icon={<LooksTwoOutlinedIcon />}
-            checkedIcon={<LooksTwoOutlinedIcon />}
+            //icon={<LooksTwoOutlinedIcon />}
```
Comment (Ari-S-123) [inline src/frontend/src/pages/GanttPage/GanttPageFilter.tsx:150]: And here.

File: src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModal.tsx [hunk 6]
```diff
@@ -147,7 +147,6 @@ const ActivateWorkPackageModal: React.FC<ActivateWorkPackageModalProps> = ({
             <NERFailButton variant="outlined" form="activate-work-package-form" onClick={onHide} sx={{ mx: 1 }}>
               Cancel
             </NERFailButton>
-
```
Comment (Ari-S-123) [inline src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModal.tsx:150]: You shouldn't have to change anything in this file for this ticket.

File: yarn.lock [hunk 7]
```diff
@@ -1492,6 +1492,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@babel/runtime@npm:^7.20.13":
```
Comment (Ari-S-123) [inline yarn.lock:1495]: I'm not sure this ticket requires any changes to `yarn.lock`

File: src/frontend/src/pages/GanttPage/GanttPageFilter.tsx [hunk 8]
```diff
@@ -47,16 +49,88 @@ const GanttPageFilter: FC<GanttPageFilterProps> = ({
...
+            }
+            checkedIcon={
+              <Box
+                sx={{
+                  display: 'flex',
+                  alignItems: 'center',
+                  border: '2px solid white',
+                  width: '2rem',
+                  height: '2rem',
+                  bgcolor: 'green'
```
Comment (Ari-S-123) [inline src/frontend/src/pages/GanttPage/GanttPageFilter.tsx:83]: Can you make the checked-off color `#ef4345` for the car checkboxes?

**Review by Ari-S-123 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (haoming5674) [reply package.json:134]: I'm not sure why that got changed, as I didn't edit anything within this file. Same for yarn.lock

**Review by anthonybernardi — APPROVED** [review]
lgtm

**Pull request merged**